### PR TITLE
Java ebs example

### PR DESCRIPTION
**Pull request by MickaelAlliel**

### CODE REVIEW

File: java-elasticbeanstalk/Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+build:
+	mkdir output && mkdir target
+	javac src/httpserver/* -d output
+	jar cvfm target/server.jar Manifest.txt -C output/ . src/*
+	wget "http://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.rookout&a=rook&v=LATEST"  -O rook.jar
+
+run:
+	export PORT=7000 && java -javaagent:rook.jar -jar target/server.jar
```
Comment (itielshwartz) [inline java-elasticbeanstalk/Makefile]: no need some more flags?

Comment (MickaelAlliel) [reply java-elasticbeanstalk/Makefile]: No it doesnt need

File: java-elasticbeanstalk/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+4. Make sure everything worked: [http://localhost:7000/?fname=rook&lname=out](http://localhost:7000/?fname=rook&lname=out)
+
+5. Go to [http://app.rookout.com](http://app.rookout.com) and start debugging! 
+
+
+## Running on AWS Elastic Beanstalk
+1. Zip the project (the files, not the parent directory)
+
+2. Upload the source bundle when creating a new Beanstalk app
```
Comment (itielshwartz) [inline java-elasticbeanstalk/README.md]: add link

File: java-elasticbeanstalk/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    $ make run
+    ```
+
+4. Make sure everything worked: [http://localhost:7000/?fname=rook&lname=out](http://localhost:7000/?fname=rook&lname=out)
+
+5. Go to [http://app.rookout.com](http://app.rookout.com) and start debugging! 
+
+
+## Running on AWS Elastic Beanstalk
+1. Zip the project (the files, not the parent directory)
```
Comment (itielshwartz) [inline java-elasticbeanstalk/README.md]: add command or makefile

File: java-elasticbeanstalk/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+## Rookout Integration explained
+
+We have added Rookout to the original project by:
+1. Adding sources to the project jar when building:
+    ```bash
+    jar cvfm target/server.jar Manifest.txt -C output/ . src/*
+    ```
+    
+2. Downloading rook javaagent available on [maven central]:
+    ```bash
```
Comment (itielshwartz) [inline java-elasticbeanstalk/README.md]: delete

File: java-elasticbeanstalk/README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
+# Quickstart for Java + Rookout and AWS Elastic Beanstalk
+
+A sample application for using Rookout + Java + AWS Elastic Beanstalk .
+
+Before following this guide we recommend reading the basic [Java + Rookout] guide
```
Comment (itielshwartz) [inline java-elasticbeanstalk/README.md]: add comment about the .ebextentions

**Pull request merged**